scripts/run-tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+MY_PATH="`dirname \"$0\"`"              # relative
+MY_PATH="`( cd \"$MY_PATH\" && pwd )`"  # absolutized and normalized
```

Comment:
```
ROOTDIR=$(git rev-parse --show-toplevel)
```

and then

```
pushd $ROOTPATH
...
popd
```

Helps to keep things a bit more clean and tidy :)

Reply:
- with this recursive path algorithm I can execute the tests even outside a git repo.
- pushd is more used to save all locations and browse them, like good alternative to subshells http://tldp.org/LDP/abs/html/subshells.html. I thing if i only move one time cd is best, less lines to do the same.

Reply:
good points.
I thought cd will also affect the path of outside the script, but apparently not, so no reason for `pushd`.
Concerning path, what do you think about this snippet (also less lines ;) )

```
DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Reply:
it's ok for me
what do you think about this?
```bash
#!/bin/bash

cd $(dirname $0)/..
export TRAVIS_BUILD_DIR=$PWD
docker-compose -f .docker/docker-compose.travis.yml run qgis /usr/src/.docker/run-docker-tests.sh
docker-compose -f .docker/docker-compose.travis.yml rm -s -f
```